framework/src/modules/chain/transactions/transactions_handlers.js

```diff
@@ -115,8 +115,17 @@ const applyGenesisTransactions = storage => async (
 		tx,
 	});
 
-	await Promise.all(transactions.map(t => t.prepare(stateStore)));
-	await Promise.all(transactions.map(t => votesWeight.prepare(stateStore, t)));
+	// eslint-disable-next-line no-restricted-syntax
```

Comment:
It would be good to have a comment explaining why we can't run in parallel and why we need two for loops here. (`to avoid someone merging this into one for loop in the future`)